### DISCUSSION TITLE
Home assistant integration to display photos on smart screens

### DISCUSSION THREAD

**Discussion by fabioteixei** [immich-app/immich#6179] [Q&A]
I want to know if it's possible to integrate Immich with home assistant on any form so I can display photos that are stored on immich on my smart screens like Google Home Hub or Echo Show?

Don't need Immich to talk with Google or Alexa, but using home assistant like a "proxy" to display the photos on those screens

**Comment by Silenoz93** [comment]
Hi,
I found following HACS component which I think does what you want:
https://github.com/outadoc/immich-home-assistant
Best regards,
